### PR TITLE
Use fontawesome everywhere...

### PR DESCRIPTION
**Pull request by vpeil**
which in fact means, remove glyphicons everywhere...

Please review! Some Icons seem to be a bit tight to the texts

### CODE REVIEW

**Comment by vpeil** [comment]
I cannot confirm. I see the icons you mentioned, and my chrome does not report any errors. Cache?

**Comment by verheyenkoen** [comment]
@phochste Probably need to rebuild your CSS using npm?

**Comment by vpeil** [comment]
@verheyenkoen can you help here? The icons are a bit tight to the text...maybe this can be done by some CSS magic?

**Comment by vpeil** [comment]
one needs the generate the forms. that solves @phochste 's problem

**Comment by verheyenkoen** [comment]
Just rebuilt and pushed the CSS file, that fixed the icon spacing problem for me.

**Comment by vpeil** [comment]
confirmed! green light from me.

what about you @phochste ?

File: views/admin/account.tt [hunk 1]
```diff
@@ -21,23 +21,23 @@
...
+	      <button type="submit" class="btn btn-default"><i><span class="fa fa-search"></span> </i>Search</button>
 	    </span>
 	  </div><!-- /input-group -->
 	</div>
       </div>
     </form>
 
     [%- IF error %]
-    <p class="text-danger">[% error %] <a href="[% uri_base %]/librecat/admin/account/edit/[% qp.id %]"><span class="glyphicon glyphicon-pencil fw"></span>Edit</a></p>
+    <p class="text-danger">[% error %] <a href="[% uri_base %]/librecat/admin/account/edit/[% qp.id %]"><span class="fa fa-pencil fw"></span>Edit</a></p>
```
[the remark this answers is not in the document]
Comment (vpeil) [reply views/admin/account.tt]: no idea, I was just searching and replacing. please feel free, to change and improve that!AA

File: scss/mypub.scss [hunk 2]
```diff
@@ -445,7 +445,7 @@ a.anchor {
 
 /* width */
 .fw,
-a span[class^='glyphicon'] {
+a span[class^='fa'] {
```
[the remark this answers is not in the document]
Comment (vpeil) [reply scss/mypub.scss]: no idea, I was just searching and replacing. please feel free, to change and improve that!AA

File: views/admin/account.tt (hunk 1 above)
[the remark this answers is not in the document]
Comment (verheyenkoen) [reply views/admin/account.tt]: I checked this and it turns out it behaves more like a uniform right margin than what fa-fw does so I guess it's ok.

**Pull request merged**